src/commands/sdk.ts

```diff
@@ -42,6 +42,13 @@ export class SdkCommands {
 			else
 				return this.runPub("get", uri);
 		}));
+		context.subscriptions.push(vs.commands.registerCommand("flutter.screenshot", async (uri) => {
+			if (!uri || !(uri instanceof Uri))
+				uri = await this.getWorkspace("Select which folder to save screenshot into");
```

Comment:
I think we should use a normal folder picker here (similar to [here](https://github.com/Dart-Code/Dart-Code/blob/76d5f6827ee432e29902f4d60560768402576dcb/src/sdk/utils.ts#L95)) so the user can put screenshots anywhere, not just in workspace roots (eg. they might want to put them on the desktop or on a screenshots sub-folder).